scripts/cmake/config/default.cmake

```diff
@@ -746,6 +753,21 @@ set(HIPPOMOCKS_URL_MD5 "6a0928dfee03fbf4c12c36219c696bae"
     "Hippomocks URL MD5 hash"
 )
 
+add_config(
```

Comment:
Change the add_config macro for GNUEFI_URL and GNUEFI_URL_MD5 to match the other variable definitions of URLs and MD5s just above